$NetBSD$

This is needed because of the use of RANDOM (possibly more)
in common.sh

--- tests/dsa/testdsa.orig	2015-07-21 09:42:08.000000000 +0000
+++ tests/dsa/testdsa
@@ -35,7 +35,7 @@ if test "${WINDIR}" != ""; then
 	exit 77
 fi 
 
-. "${srcdir}/../scripts/common.sh"
+bash "${srcdir}/../scripts/common.sh"
 
 PORT="${PORT:-$RPORT}"
 
